app/views/steps/case/codefendants/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+
+    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
+
+    <%= step_form @form_object do |f| %>
+      <%= f.fields_for :codefendants do |c| %>
+        <%= c.govuk_fieldset legend: { text: t('.codefendant_legend', index: c.index + 1) } do %>
```

Comment:
Was displaying the codefendant with index discussed? It's not part of the designs

Reply:
Font size for the legend in the design also looks to be the same size as the field label.  `size: 's'` gives us 1 rem which I think is the same

Reply:
I think it is necessary if you want to make this page perform well in terms of accessibility. Otherwise it would be a series of repetitions not meaning anything to a screen reader. Also, facilitates the errors in the summary, which are read aloud first thing if an error happens.

And finally, in the check your answers they are indexed, which as a user would expect to match the same indexing in the page where I've entered the names.

Reply:
Re: legend font size. I'm being consistent with same sizing we already had in the legends in client personal details (Full name, Other names, Date of birth).

I think as it indicates a degree of hierarchy it makes sense to be slightly bigger in size, and in fact we are using the default size provided by the form builder so provable best to stick the the defaults IMO.